### PR TITLE
Make gtfo show the sender

### PR DESCRIPTION
**Pull request by Scuph**
In the broadcast message and the kick message

### CODE REVIEW

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 1]
```diff
@@ -98,7 +99,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         TFM_BanManager.addUuidBan(new TFM_Ban(TFM_UuidManager.getUniqueId(player), player.getName(), sender.getName(), null, reason));
 
         // kick Player:
-        player.kickPlayer(ChatColor.RED + "GTFO" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : ""));
+        player.kickPlayer(ChatColor.RED + "GTFO - " + (reason != null ? (ChatColor.YELLOW + "Banned by " + ChatColor.RED + sender.getName() + ChatColor.YELLOW + ". " + "\nReason: " + reason) : ""));
```
Comment (TheRevalation) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: (reason != null ? might have to be after (ChatColor.YELLOW + "Banned by "

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 2]
```diff
@@ -80,7 +80,8 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         final StringBuilder bcast = new StringBuilder()
                 .append(ChatColor.RED)
-                .append("Banning: ")
+                .append(sender_p)
+                .append(" - Banning: ")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: You're simulating `adminAction()` behavior, use the proper method please.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 3]
```diff
@@ -98,7 +99,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         TFM_BanManager.addUuidBan(new TFM_Ban(TFM_UuidManager.getUniqueId(player), player.getName(), sender.getName(), null, reason));
 
         // kick Player:
-        player.kickPlayer(ChatColor.RED + "GTFO" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : ""));
+        player.kickPlayer(ChatColor.RED + "GTFO - " + (ChatColor.YELLOW + "Banned by " + ChatColor.RED + sender.getName() + ChatColor.YELLOW + ". " + (reason != null ? "\nReason: " + reason) : ""));
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: Missing a newline perhaps.

Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: I think a newline after "GTFO" would be in order.

**Comment by TheRevalation** [comment]
@JeromSar You want me to use the adminaction method? Ok..

**Comment by xDestroyer217** [comment]
@TheRevalation, It really likely is the same thing.... lol.

**Comment by TheRevalation** [comment]
@xDestroyer217 I thought that, @JeromSar why not just leave it the way I changed it, it works.

**Comment by StevenLawson** [comment]
It's easy to write crap code that "works", the key is to write code in a manner that is consistent with the rest of the code base. For this reason, do not duplicate code that already has methods that do the same thing.

**Comment by TheRevalation** [comment]
@StevenLawson All it's adding is the damn sender name in the broadcast message -

OLD: 'Banning: player and IPs: *.*.*.*'
NEW: Admin - Banning: player IPs: *.*.*.*

Both ways work, but if both work, why does it matter which one we use, it causes no confusion?

**Comment by JeromSar** [comment]
@Scuph If we ever change the way admin broadcasts are displayed, this will leave /gtfo unchanged. Your code isn't future-proof.

**Comment by JeromSar** [comment]
Closing for inactivity.

**Comment by Wild1145** [comment]
@JeromSar This is a PR not an issue ;) :P

**Comment by JeromSar** [comment]
@Wild1145 Herp derp fixed.

**Pull request closed without merging**